doc/developers-guide.md

```diff
@@ -164,6 +164,46 @@ Now you can rebuild `confd`, just as described above, and restart Infix:
     make confd-rebuild all run
 
 
+### `statd`
+
+The Infix `src/statd/` (Status Daemon) populates operational data in 
+the sysrepo datastore using again XPath subscriptions. 
+
+StatD relies entirely on `yanger`, a Python script that gathers data from 
+local linux services and feeds it into sysrepo.
+
+To apply changes, rebuild the image:
+
+    make python-statd-rebuild statd-rebuild all
+
+Rebuilding statd for every change during development process can be 
+tedious. Instead, `yanger` allows remote execution, running the script
+directly on the host system (test container):
+
+    infamy0:test # ../src/statd/python/yanger/yanger -x "../utils/ixll -A ssh d3a" ieee802-dot1ab-lldp
```

Comment:
You need another paragraph to explain what `ixll` does and how it ties together with yanger.